docs/UserGuide.md

```diff
@@ -326,32 +329,35 @@ Example:
 
 ### **Profile**
 
-Note: You must be on the **Profile** tab in order to execute these commands.
+The profile feature offers you the ability to keep track and maintain your personal portfolio. It stores
+profile items which are entities that describe an achievement, a past experience or a skill you have acquired.
+It assists you in crafting a tailored resume for your internship applications by allowing you to find items in your
+profile which are relevant to the job description through the [find feature](#finding-items-in-profile-find-me).
 
-Your profile can contain 3 categories of information.
+Each profile item belongs to  1 of 3 categories.
 
 Valid `CATEGORY` specifiers:
 - `achievement`
 - `experience`
 - `skill`
 
-Each profile item (bit of information) also contains a `TITLE` that describes the item, and you can optionally 
+Each profile item also contains a `TITLE` that describes the item, and you can optionally 
 add additional `DESCRIPTOR`s to further describe the item in point form.
 
 #### Adding item to profile: `add me`
 
-Adds experience, skills or achievements descriptors to your profile.
+Adds a profile item to your profile.
 
 Format: `add me c/CATEGORY t/TITLE [d/DESCRIPTOR]...`
 
-* Category specifies a category which can be either experience, skills or achievement.
+* `CATEGORY` specifies a category which can be either experience, skills or achievement.
 
 Examples:
-* `add me c/experience t/Internship at Govtech d/Implemented automate testing 
-using TravisCI d/Implemented dashboard to track code coverage`
-* `add me c/achievement t/special recognition in Hack n Roll`
 * `add me c/skill t/HTML d/Learn how to create divs`
-
+* `add me c/achievement t/special recognition in Hack n Roll`
+* For example, executing `add me c/experience t/Internship at Govtech d/Implemented automate testing 
+using TravisCI d/Implemented dashboard to track code coverage` on an empty profile will add the profile item as follows: <br />
+![AddProfileSS](images/AddProfileItemSS.PNG)
```

Comment:
it should be .png, I think images are case-sensitive.

Reply:
The images turn out well in my markdown on intellij. Also the images in company all have this casing too

Reply:
oh i think intellij does care about case-sensitive but github does haha
![Uploading Screenshot 2020-10-24 at 7.14.34 PM.png…]()